deployment/helm/node-feature-discovery/values.yaml

```
@@ -469,6 +469,12 @@ worker:
 
   priorityClassName: ""
 
+  # updateStrategy:
+  #   type: RollingUpdate
+  #   rollingUpdate:
+  #     maxSurge: 10%
+  #     maxUnavailable: 10%
```

Depending on how the template is written, we might need

```suggestion
  updateStrategy: {}
    # type: RollingUpdate
    # rollingUpdate:
    #  maxSurge: 10%
    #  maxUnavailable: 10%
```
ALTERNATIVELY, we code put defaults, here, too

```suggestion
  updateStrategy: {}
    type: RollingUpdate
    rollingUpdate:
      maxSurge: 1
      maxUnavailable: 1
```